code/modules/hydroponics/grown/banana.dm

```diff
@@ -109,6 +109,39 @@
 	desc = "A peel from a bluespace banana."
 	icon_state = "banana_peel_blue"
 
+//BANANIUM
+
+/obj/item/seeds/banana/bananium
+	name = "pack of Bananium seeds"
+	desc = "They're seeds that grow into bananium trees. When grown, keep away from robotics."
+	icon_state = "seed-banana"
+	species = "bananium"
+	plantname = "Bananium Tree"
+	product = /obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium
+	lifespan = 7
+	endurance = 1
+	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
+	icon_dead = "banana-dead"
+	genes = list(/datum/plant_gene/trait/slip)
+	mutatelist = list()
+	reagents_add = list("vitamin" = 0.04, "nutriment" = 0.02)
+	rarity = 50 //Nanotrason would love this
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium
+	seed = /obj/item/seeds/banana/bananium
+	name = "Bananium Banana"
+	desc = "If this plant has enough potency, there will be something special inside...."
+	icon_state = "banana"
+	item_state = "banana"
+	trash = /obj/item/weapon/grown/bananapeel
+	filling_color = "#FFFF00"
+	bitesize = 5
+	
+/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/add_juice()
```

Comment:
Try this:
`/obj/item/weapon/reagent_containers/food/snacks/grown/banana/bananium/New()`
`	materials = list(MAT_BANANIUM = potency*20)`
`	..()`